### PR TITLE
update missing probe severity to info

### PR DESCRIPTION
**Pull request by juanvallejo**
Fixes https://github.com/openshift/origin/issues/10294

This patch changes the marker severity of missing probes (liveness and readiness) from `osgraph.WarningSeverity` to `osgraph.InfoSeverity`. It also adds a linebreak between `Errors`, `Warnings`, and `Infos`.

**Example**
```
$ oc status -v
In project default on server https://10.13.137.149:8443

http://www.pictre.org (svc/frontend)

https://idling-echo-reencrypt-default.router.default.svc.cluster.local (reencrypt) (svc/idling-echo)
http://idling-echo-default.router.default.svc.cluster.local
  dc/idling-echo deploys docker.io/openshift/node:latest
    deployment #1 deployed 2 days ago - 3 pods

svc/kubernetes - 172.30.0.1 ports 443, 53->8053, 53->8053

dc/test-deployment-config deploys docker.io/openshift/origin-pod:latest
  deployment #1 deployed 2 days ago - 1 pod

bc/test-buildconfig source builds git://github.com/openshift/ruby-hello-world.git on docker.io/centos/ruby-22-centos7:latest
  not built yet

Errors:
  * route/foo is routing traffic to svc/foo, but either the administrator has not installed a router or the router is not selecting this route.
    try: oc adm router -h
  * route/idling-echo is routing traffic to svc/idling-echo, but either the administrator has not installed a router or the router is not selecting this route.
    try: oc adm router -h
  * route/idling-echo-reencrypt is routing traffic to svc/idling-echo, but either the administrator has not installed a router or the router is not selecting this route.
    try: oc adm router -h
  * route/my-route is routing traffic to svc/frontend, but either the administrator has not installed a router or the router is not selecting this route.
    try: oc adm router -h

Warnings:
  * route/foo is supposed to route traffic to svc/foo but svc/foo doesn't exist.
  * route/idling-echo doesn't have a port specified and is routing traffic to svc/idling-echo which uses multiple ports.
  * route/idling-echo-reencrypt doesn't have a port specified and is routing traffic to svc/idling-echo which uses multiple ports.

Infos:
  * dc/idling-echo has no readiness probe to verify pods are ready to accept traffic or ensure deployment is successful.
    try: oc set probe dc/idling-echo --readiness ...
  * dc/idling-echo has no liveness probe to verify pods are still running.
    try: oc set probe dc/idling-echo --liveness ...
  * dc/test-deployment-config has no readiness probe to verify pods are ready to accept traffic or ensure deployment is successful.
    try: oc set probe dc/test-deployment-config --readiness ...
  * dc/test-deployment-config has no liveness probe to verify pods are still running.
    try: oc set probe dc/test-deployment-config --liveness ...

View details with 'oc describe <resource>/<name>' or list everything with 'oc get all'.
```

cc @jwforres @deads2k @fabianofranz

### CODE REVIEW

**Comment by juanvallejo** [comment]
[test]

**Comment by jwforres** [comment]
I would use "Info:" for the header not "Infos:"  you wouldn't say
Informations, you would say Information

**Comment by juanvallejo** [comment]
@jwforres 
> I would use "Info:" for the header not "Infos:"  you wouldn't say
Informations, you would say Information

Done! Do you think `Notices` or something along those lines would sound better instead of `Info`?

**Comment by jwforres** [comment]
Hmm that would probably be OK too, I don't have a strong opinion either way.

File: pkg/cmd/cli/describe/projectstatus.go [hunk 1]
```diff
@@ -304,6 +304,10 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 		warningMarkers := allMarkers.BySeverity(osgraph.WarningSeverity)
 		if len(warningMarkers) > 0 {
 			if d.Suggest {
+				// add linebreak between Errors list and Warnings list
+				if len(errorMarkers) > 0 {
+					fmt.Fprintln(out, "")
```
Comment (fabianofranz) [inline pkg/cmd/cli/describe/projectstatus.go]: nit, `fmt.Fprintln(out)`

File: pkg/cmd/cli/describe/projectstatus.go [hunk 2]
```diff
@@ -322,6 +326,31 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 			}
 		}
 
+		infoMarkers := allMarkers.BySeverity(osgraph.InfoSeverity)
+		if len(infoMarkers) > 0 {
+			if d.Suggest {
+				// add linebreak between Warnings list and Info List
+				if len(warningMarkers) > 0 || len(warningMarkers) > 0 {
+					fmt.Fprintln(out, "")
```
Comment (fabianofranz) [inline pkg/cmd/cli/describe/projectstatus.go]: Same here.

**Comment by fabianofranz** [comment]
LGTM, no strong opinion about the wording. Let me know when this is good to go.

**Comment by juanvallejo** [comment]
@fabianofranz thanks for the review, almost good to go, added a `printMarkerSuggestions` function to avoid duplicating code for `Errors`, `Warnings`, and `Info`

**Comment by juanvallejo** [comment]
[test]

**Comment by juanvallejo** [comment]
integration check flaked on https://github.com/openshift/origin/issues/9203 re[test]

File: pkg/cmd/cli/describe/projectstatus.go [hunk 3]
```diff
@@ -282,44 +282,31 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
...
-						fmt.Fprintln(out, indent+"  try: "+s)
-					}
+			printMarkerSuggestions(warningMarkers, d.Suggest, out, indent)
+		}
+
+		infoMarkers := allMarkers.BySeverity(osgraph.InfoSeverity)
+		if len(infoMarkers) > 0 {
+			if d.Suggest {
+				// add linebreak between Warnings list and Info List
+				if len(warningMarkers) > 0 || len(warningMarkers) > 0 {
```
Comment (fabianofranz) [inline pkg/cmd/cli/describe/projectstatus.go]: Double-check just in case.  :)

Comment (juanvallejo) [inline pkg/cmd/cli/describe/projectstatus.go]: :) Thanks for catching that

**Comment by fabianofranz** [comment]
LGTM [merge]

**Pull request merged**